### ISSUE TITLE
Importing all findings to a docx template

### ISSUE THREAD

**Issue by Michael-Aguilera** [DefectDojo/django-DefectDojo#2202]
I've been trying to find a way to import the data from the asciidoc report into my docx template I use for Pentesting and have yet to find a way to do so! Which is why I am here :)... It would be a great feature to import all the findings in an engagement to a docx report. A good tool to reference here is Serpico as they did a good job with importing results to a docx template.

**Comment by xNoJustice** [comment]
Same like that? 
![Screenshot_1](https://user-images.githubusercontent.com/45815512/79476722-06cbaf00-8012-11ea-9b7e-d4ab2b4ece7a.png)
![Screenshot_4](https://user-images.githubusercontent.com/45815512/79476728-07fcdc00-8012-11ea-8d13-8097caf1b6ac.png)

**Comment by Michael-Aguilera** [comment]
This is a feature I am really looking for on the reporting side of things...

More like this (keep note that this only lists where the client details and author of the report details):
![image](https://user-images.githubusercontent.com/22526586/79484311-bb1a0500-8013-11ea-8e9b-23c4e3d0a6ae.png)

Here is how the vulnerability details are inserted:

![image](https://user-images.githubusercontent.com/22526586/79484736-66c35500-8014-11ea-93f0-52d8f3d8eab6.png)

where it has variables to insert to specific areas of a report:

![image](https://user-images.githubusercontent.com/22526586/79484458-f4527500-8013-11ea-9905-238ebcc51eee.png)

Findings are imported into the report like so:
![image](https://user-images.githubusercontent.com/22526586/79484843-970af380-8014-11ea-955e-9f51888c211d.png)

And vulnerabilities are imported to the report like so:
![image](https://user-images.githubusercontent.com/22526586/79484974-c91c5580-8014-11ea-9b8a-0f848ef0fed9.png)

**Comment by dayal18** [comment]
Yeah.. we are still using Serpico for reporting and having that feature in Dojo would be awesome.

**Comment by Demaz93** [comment]
I create a full python tool like Serpico but with a better template engine for my pentest reporting.
Personally I used docxtpl ([https://docxtpl.readthedocs.io/](https://docxtpl.readthedocs.io/)) library, that use jinja2 for tamplating docx files, easyest syntax than Serpico.
